docs/tools/swift.pod

```diff
@@ -1,30 +1,114 @@
 =pod
 
+=encoding utf8
+
 =head1 NAME
 
-swift - the amazingly new programming language
+swift - General purpose programming language
```

Comment:
Below, the spelling is "general-purpose"; here, it is "general purpose"--you'll need to pick one.

May I also suggest following the example of Python, Ruby, LLDB, and others in using two-dashes (i.e., "swift -- General-purpose programming language")?